CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+### citus v8.2.0 (March 28, 2019) ###
```

Comment:
Should we mention about drop for 9.6 support? https://github.com/citusdata/citus/pull/2554